tests/integ/test_data_source_api.py

```diff
@@ -111,26 +110,26 @@ def test_dbapi_with_temp_table(session, caplog):
 
 def test_dbapi_oracledb(session):
     df = session.read.dbapi(
-        oracledb_create_connection,
+        create_connection_oracledb,
         table=ORACLEDB_TABLE_NAME,
         max_workers=4,
         query_timeout=5,
-    )
-    assert df.collect() == oracledb_all_type_data_result
+    ).order_by("ID")
```

Comment:
curious why we have `order_by` here?
without partitioning column, we don't do parallel processing right?